text/0000-localizing-api-messages.md

```diff
@@ -0,0 +1,135 @@
+
+- Start Date: 2022-03-29
+- RFC PR: 
+- FOLIO Issue: 
+- Submitter : Radhakrishnan Gopalakrishnan (rgopalakrishnan@ebsco.com)
+- Co-Submitter(s) : Zak Burke (zburke@ebsco.com)
+- Sub Group :
+
+# Localizing API (Backend) Messages
+
+## Summary
+  The purpose of this RFC is to define an approach that allows API developer to provide
+messages in the API response depending on the language preference set in the request. 
+
+
+## Motivation
+
+- Help users understand the information coming from the application backend using a language they understand better
+- To allow FOLIO platform to be used by users across the world 
+- To create a tremendous opportunity for growth that we can never could have achieved within just one country.
```

Comment:
Help translators by standardizing keys and syntax (plural syntax, etc.).
Help translation devops by standardizing the location of translation files in the repositories and their format.

Reply:
There was a debate initially about the scope of this RFC and the level on detail/directions we needed to provide. So, interest of minimizing the scope and being less prescriptive about implementation, these were left out. I certainly amenable to adding this back in.

Reply:
Added